packages/fx-core/src/plugins/resource/bot/index.ts

```diff
@@ -145,6 +145,45 @@ export class TeamsBot implements Plugin {
     );
   }
 
+  private wrapError(
+    e: any,
+    context: PluginContext,
+    sendTelemetry: boolean,
+    name: string
+  ): FxResult {
+    if (e.innerError) {
+      e.message += ` Detailed error: ${e.innerError.message}.`;
+      if (e.innerError.response?.data?.errorMessage) {
+        e.message += ` Reason: ${e.innerError.response?.data?.errorMessage}`;
+      }
+    }
+    Logger.error(e.message);
```

Comment:
I think solution will log the e.message since we explicitly return err(e). Please make sure there are no duplicated log. In my opinion, we only need to log the innerError in plugin.